src/mpid/common/thread/mpidu_thread_fallback.h

```diff
@@ -249,7 +203,7 @@ M*/
             MPL_thread_id_t self_;                                      \
             MPL_thread_self(&self_);                                    \
             MPL_thread_same(&self_, &mutex.owner, &equal_);             \
-            if (!equal_) {                                               \
+            if (!equal_) {                                              \
```

Comment:
Extra space added.

Reply:
Extra space *deleted*. It was extra before.